spec/design_topics/copies_views_and_mutation.rst

```diff
@@ -3,6 +3,11 @@
 Copy-view behaviour and mutability
 ==================================
 
+.. admonition:: Mutating views
+   :class: important
+
+   Array API consumers are *strongly* advised to avoid *any* mutating operations when an array object may be a "view" (i.e., an array whose data refers to memory that belongs to another array). This admonition may become more strict in the future (e.g., this specification may require that view mutation be prohibited and trigger an exception). Accordingly, only perform mutation operations (e.g., in-place assignment) when absolutely confident that array data belongs to one, and only one, array object.
```

Comment:
`may be a "view"` -> `may be a "view" ... or contains data which is used in another array.` Maybe rephrase a little if this is awkward; the point is important though. Whether an array _is a view_ is easy to determine typically, while whether an array _owns memory viewed by another array_ is much more difficult.

Reply:
Updated to include the proposed changes.